en/mavgen_c/README.md

```diff
@@ -34,17 +34,34 @@ This will automatically add the header files for all messages in your dialect, a
 > **Tip** *Do not include the individual message files*.
   If you generate your own headers, you will have to add their output location to your C compiler's search path.
 
-When compiling the project, we recommend that you specify the top-level output directory AND all generated dialects and versions (this will give the greatest compatibility with existing code and examples):
+When compiling the project, make sure to add the include directory:
 ```sh
-$ gcc ... -I generated/include -I generated/include/common ...
+$ gcc ... -I generated/include ...
 ```
 
-In order to *send messages* you will also need to declare a variable `mavlink_system` in the **global scope**, specifying the system id and component ID of your component:
-```cpp
```

Comment:
I am assuming that people don't need this, or that it appears elsewhere, or is untrue.